cli/index.js

```diff
@@ -1,7 +1,9 @@
 #!/usr/bin/env node
 
 const chalk = require("chalk");
-const { isSupportedNodeVersion } = require("../utils/versionUtils");
+
+// TODO: How to link lerna with unpublished package?
```

Comment:
@andreyluiz any idea how to fix this?

Reply:
You need to add the dependency for the `clio-utils@0.1.0` package on the clio project `package.json`. Npm will automagically use lerna to know where to get the package.